### DISCUSSION TITLE
Not support ":where(&)" selector.

### DISCUSSION THREAD

**Discussion by rxy001** [vanilla-extract-css/vanilla-extract#1415] [Ideas]
Error "Invalid selector: :where(&)" when used :where(&) to reduce specificity. 

I think this is really useful when used component variants to override the default styles.

**Comment by plesiecki** [comment]
a workaround btw
```ts
export const rule = style({});

globalStyle(`:where(${rule})`, {});
```

**Reply by tomahl** [reply]
See here [PR](https://github.com/vanilla-extract-css/vanilla-extract/pull/1702)
